packages/p2p/src/components/orders/order-table/order-table.jsx

```diff
@@ -27,6 +27,19 @@ const OrderTable = ({ showDetails }) => {
     const is_active_tab = general_store.order_table_type === order_list.ACTIVE;
     return (
         <React.Fragment>
+            {general_store.banner_config.reduced_order_time && (
```

Comment:
Why are we checking this if it's always true?

Reply:
If this is a temporary feature, there is no need for separate component